### PR TITLE
Upgrade kubernetes versions for testing

### PR DESCRIPTION
**Pull request by simonswine**
```release-note
NONE
```

### CODE REVIEW

File: puppet/modules/tarmak/Makefile [hunk 1]
```diff
@@ -1,46 +1,66 @@
 BUNDLE_DIR ?= .bundle
 
+VERSION_1_10 := 1.11.0
```
Comment (MattiasGees) [inline puppet/modules/tarmak/Makefile]: Shouldn't this be 1_11?

Comment (simonswine) [reply puppet/modules/tarmak/Makefile]: Good point!

**Pull request merged**